README.md

```diff
@@ -60,21 +63,20 @@ If it is not, you can apply for whitelisting at hello@beamerbridge.com.
          tar xz -C data --strip-components=1 beamer-1.0.3/deployments
     ```
 
-    The above command will download Beamer contracts' deployment information which the
-    agent needs in order to run.
+   The above command will download Beamer contracts' deployment information which the
+   agent needs in order to run.
 
 1. Copy your JSON keystore file to `data/account`.
 
-
 ### Mainnet configuration
 
 1. Edit `data/agent-mainnet.conf` and make sure that the following keys have correct values:
 
-   - `[account.path]` - the path to your JSON keystore file
-   - `[account.password]` - the password to unlock your keystore file
-   - `[chains.l1.rpc-url]` - the RPC endpoint to use for mainnet Ethereum (e.g. `https://mainnet.infura.io/v3/ID`)
-   - `[chains.arbitrum.rpc-url]` - the RPC endpoint to use for mainnet Arbitrum (e.g. `https://arb1.arbitrum.io/rpc`)
-   - `[chains.optimism.rpc-url]` - the RPC endpoint to use for mainnet Optimism (e.g. `https://mainnet.optimism.io`)
+    - `[account.path]` - the path to your JSON keystore file
+    - `[account.password]` - the password to unlock your keystore file
+    - `[chains.l1.rpc-url]` - the RPC endpoint to use for mainnet Ethereum (e.g. `https://mainnet.infura.io/v3/ID`)
```

Comment:
the key for l1 is now called `base-chain`

Reply:
Then I think the `agent.conf.example` in the main beamer repo is also outdated.